packages/react/src/primitives/Tabs/Tabs.tsx

```diff
@@ -72,20 +76,12 @@ const TabsPrimitive: Primitive<TabsProps, typeof Flex> = (
         </Flex>
       </List>
       {React.Children.map(nonNullChildren, (child, index) => {
-        if (isTabsType(child)) {
+        if (isExtendedTabItem(child)) {
           return (
             <Panel key={index} value={`${index}`}>
               {child.props.children}
             </Panel>
           );
-        } else {
-          // at this point the child defined (not null or undefined)
-          // it is NOT a TabItem, so log a message
-          if (child) {
-            console.warn(
```

Comment:
I know you explained this, but why are we not logging a message anymore?

Reply:
It was very easy to bypass the `isTabsType` predicate, so the warning here was also fragile